eyewear_shop/data/pos_config.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="pos_config_eyewear_shop" model="pos.config">
+        <field name="name">Eyewear Shop</field>
+        <field name="iface_tipproduct" eval="True"/>
+        <field name="iface_customer_facing_display_local" eval="True"/>
+        <field name="iface_print_auto" eval="True"/>
+        <field name="iface_customer_facing_display" eval="True"/>
+        <field name="ship_later" eval="True"/>
+        <field name="limit_categories">True</field>
+        <field name="iface_available_categ_ids" eval="[(6, 0, [
+            ref('pos_category_1'),
+            ref('pos_category_2'),
+            ref('pos_category_3'),
+        ])]"/>
```

Comment:
Wasn't it only necessary if we wanted to split categories between different pos.config? It should work without that
```suggestion
```